tutorials/autotvm/tune_relay_x86.py

```diff
@@ -256,6 +274,29 @@ def tune_and_evaluate(tuning_opt):
 #    [Task 10/12]  Current/Best:  182.33/1734.45 GFLOPS | Progress: (360/360) | 1755.06 s Done.
 #    [Task 11/12]  Current/Best:  372.18/1745.15 GFLOPS | Progress: (360/360) | 1684.50 s Done.
 #    [Task 12/12]  Current/Best:  215.34/2271.11 GFLOPS | Progress: (400/400) | 2128.74 s Done.
+#    INFO Start to benchmark layout transformation...
+#    INFO Benchmarking layout transformation successful.
+#    INFO Start to run dynamic programming algorithm...
+#    INFO Start forward pass...
+#    INFO Finished forward pass.
+#    INFO Start backward pass...
+#    INFO Finished backward pass...
+#    INFO Finished DPExecutor run.
+#    INFO Writing optimal schedules to resnet-18_graph_opt.log successfully.
+#
+#    Evaluation of the network compiled in 'default' mode without auto tune:
+#    Compile...
+#    Evaluate inference time cost...
+#    Mean inference time (std dev): 4.5 ms (0.03 ms)
+#
+#    Evaluation of the network been tuned on kernel level:
+#    Compile...
+#    Evaluate inference time cost...
+#    Mean inference time (std dev): 3.2 ms (0.03 ms)
+#
+#    Evaluation of the network been tuned on graph level:
 #    Compile...
+#    Config for target=llvm -keys=cpu -link-params=0, workload=('dense_nopack.x86', ('TENSOR', (1, 512), 'float32'), ('TENSOR', (1000, 512), 'float32'), None, 'float32') is missing in ApplyGraphBest context. A fallback configuration is used, which may bring great performance regression.
+#    Config for target=llvm -keys=cpu -link-params=0, workload=('dense_pack.x86', ('TENSOR', (1, 512), 'float32'), ('TENSOR', (1000, 512), 'float32'), None, 'float32') is missing in ApplyGraphBest context. A fallback configuration is used, which may bring great performance regression.
```

Comment:
Do we know the root cause of these warnings?

Reply:
Good question. These warnings appear after graph level tuning. I can only suppose that graph optimizer decided to change layout that led to selection of different kernels which were not optimized on the previous stage